notebooks/Emukit-constrained-optimization.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>Would be nice to break this code down into few cells, with a short text description for each</p>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>